### DISCUSSION TITLE
Option to Show All Files in Notes View

### DISCUSSION THREAD

**Discussion by quinn-p-mchugh** [damionx7/Zettel-Notes-Documentation#62] [Feature Requests]
Copying from [Canny](https://zettel-notes.canny.io/feature-requests/p/option-to-show-all-files-in-notes-view):

> Add an option to list all files in a folder in the notes view, not just the markdown / text files. This would essentially act as a browser where non-note files are opened in the default external application.
> 
> Right now, you can access these files via the gallery, but that does not keep them in their hierarchical structure or in the context of the notes around them.

Similar requests: #61 and #63

**Comment by damionx7** [comment]
Merged with https://github.com/damionx7/Zettel-Notes-Documentation/discussions/70.
